add_new/upload.php

```diff
@@ -16,3 +16,4 @@
 
 $query->execute();
 
+echo 'Your new project has been successfully uploaded!';
```

Comment:
how do you know it was successful? you need to check on the result of execute function call